src/components/chips/js/chipsController.js

```diff
@@ -0,0 +1,86 @@
+(function () {
+  'use strict';
+  angular
+      .module('material.components.chips')
+      .controller('MdChipsCtrl', MdChipsCtrl);
+
+  function MdChipsCtrl ($scope, $element, $q, $mdUtil, $mdConstant, $log) {
+    //-- private variables
+    var self = this;
+    this.$mdConstant = $mdConstant;
+
+    /** @type {!angular.NgModelController} */
+    this.ngModelCtrl = null;
+
+    /** @type {Array.<Object>} */
+    this.items = [];
+
+    //-- public variables
+    self.scope = $scope;
+
+    return self.init();
+
+    // How to update the model:
+    // self.ngModelCtrl.$setViewValue(new Date(Number(this.dataset.timestamp)));
+    // self.ngModelCtrl.$render();
+
+  }
+
+  /** 
+   * The model used by the input element.
+   * @type {!String}
+   */
+  MdChipsCtrl.prototype.itemBuffer = '';
```

Comment:
Since `itemBuffer` is a string, this shouldn't be able to cause any issues since changing its value will assign the new value to the instance, not the prototype.